pkg/cmd/cli/cmd/deploy.go

```diff
@@ -26,6 +26,12 @@ NOTE: This command is still under active development and is subject to change.`
 	deploy_example = `  // Display the latest deployment for the 'database' deployment config
   $ %[1]s deploy database
 
+  // List all deployments for the 'database' deployment config
+  $ %[1]s deploy database -L
```

Comment:
> On May 7, 2015, at 8:19 PM, Fabiano Franz <notifications@github.com> wrote:
> 
> In pkg/cmd/cli/cmd/deploy.go:
> 
> > @@ -26,6 +26,12 @@ NOTE: This command is still under active development and is subject to change.`
> >  	deploy_example = `  // Display the latest deployment for the 'database' deployment config
> >    $ %[1]s deploy database
> >  
> > +  // List all deployments for the 'database' deployment config
> > +  $ %[1]s deploy database -L
> This one will list the history of deployments for the given dc, while describe only gives you the latest one. Also extends the functionality that was already available in the command when running without flags. Sample output:
> 
Why doesn't describe give me that?  Why can't describe show me historical data?  Why do we need two?
> All 6 deployments for test1/database:
> 
>   #6 deployed about an hour ago
>   #5 deployed about an hour ago
>   #4 deployed about an hour ago
>   #3 deployed 2 hours ago
>   #2 deployed 2 hours ago
>   #1 deployed 2 hours ago
> I do think though we could add more information to the output (or/and add a --verbose mode). What would you guys suggest?
> 
> —
> Reply to this email directly or view it on GitHub.
>